x-pack/test/api_integration/apis/entity_manager/definitions.ts

```diff
@@ -38,5 +45,53 @@ export default function ({ getService }: FtrProviderContext) {
         await uninstallDefinition(supertest, mockBackfillDefinition.id);
       });
     });
+    describe('entity data', () => {
+      let dataForgeConfig: PartialConfig;
+      let dataForgeIndices: string[];
+
+      before(async () => {
+        dataForgeConfig = {
+          indexing: {
+            dataset: 'fake_stack',
+            eventsPerCycle: 100,
+            interval: 60_000,
+          },
+          schedule: [
+            {
+              template: 'good',
+              start: 'now-15m',
+              end: 'now+5m',
+            },
+          ],
+        };
+        dataForgeIndices = await generate({ client: esClient, config: dataForgeConfig, logger });
+        await waitForDocumentInIndex({
+          esClient,
+          indexName: 'kbn-data-forge-fake_stack.admin-console-*',
+          docCountTarget: 2020,
+          retryService,
+          logger,
+        });
+      });
+
+      after(async () => {
+        await esDeleteAllIndices(dataForgeIndices);
+        await uninstallDefinition(supertest, mockDefinition.id, true);
+        await cleanup({ client: esClient, config: dataForgeConfig, logger });
+      });
+
+      it('should create the proper entities in the latest index', async () => {
+        await installDefinition(supertest, mockDefinition);
+        const sample = await waitForDocumentInIndex({
```

Comment:
out of curiosity is the retry count or timeout configurable ?

Reply:
These were constants at the top of the file, I changed them to options where the constants are the default values.